templates/travis/.travis/release.sh.j2

```diff
@@ -0,0 +1,59 @@
+if [ $# -eq 0 ]; then
+    echo "No arguments provided"
+    exit 1
+fi
+
+export PLUGIN_DIR={{ plugin_name }}
+export BASEDIR=$(dirname $(readlink -f "$0"))
+export NEW_VERSION=$1
+export DEV_VERSION=$VERSION\.dev
+
+cd $BASEDIR/..
+
+towncrier --version $NEW_VERSION --draft
```

Comment:
I wouldn't bother with this. Once we fully automate this and move it to a CI, we won't be able to run this step.

I think the release nanny will have to run this before triggering the release.

Reply:
I agree. Those changelog entries probably need to be revisited redactorially anyway.